harvestingkit/elsevier_package.py

```diff
@@ -638,6 +638,9 @@ def get_publication_date(self, xml_doc):
                     )
                     return start_date.strftime("%Y-%m-%d")
             import dateutil.parser
+            #dateutil.parser.parse cant process dates like April-June 2016
+            start_date = re.sub('([A-Z][a-z]+).*[A-Z][a-z]+ (\d{4})', 
```

Comment:
that's a fairly generic regexp with the .* potentially matching a very long sequence. is it possible to be more specific based on the actual patterns you have seen in the harvest? e.g. 
```
re.sub('([A-Z][a-z]+)[\s-]+[A-Z][a-z]+ (\d{4})
```